kpm/lib/kpm/base_artifact.rb

```diff
@@ -37,7 +37,7 @@ class BaseArtifact
     class << self
       def pull(logger, group_id, artifact_id, packaging='jar', classifier=nil, version='LATEST', destination_path=nil, sha1_file=nil, force_download=false, verify_sha1=true, overrides={}, ssl_verify=true)
         coordinate_map = {:group_id => group_id, :artifact_id => artifact_id, :packaging => packaging, :classifier => classifier, :version => version}
-        pull_and_put_in_place(logger, coordinate_map, destination_path, is_ruby_plugin_and_should_skip_top_dir(group_id, artifact_id), sha1_file, force_download, verify_sha1, overrides, ssl_verify)
```

Comment:
This is actually a regression. I was planning on moving it to the `KillbillPluginArtifact` class